### PR TITLE
Add proxy server code and release workflow

### PR DESCRIPTION
**Pull request by ThumulaPerera**
## Purpose
$subject

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    getProxyUrl,
+    getLocalAppUrl,
+}
+
+const choreoProxy = createProxyMiddleware({
+    target: configs.getChoreoAppUrl(),
+    changeOrigin: true,
+    secure: false,
+    onProxyReq: (proxyReq) => {
+        if (proxyReq.path.startsWith('/auth')) {
```
Comment (senthalan) [inline server.js:89]: shall we add a log line with the request URL?

So when troubleshooting the developers will know that the request is going through the proxy

**Comment by senthalan** [comment]
Shall we also add a readme with a small explanation of this proxy and link to the doc

**Comment by ThumulaPerera** [comment]
> Shall we also add a readme with a small explanation of this proxy and link to the doc

Updated the readme with [Update Readme](https://github.com/wso2/choreo-managed-auth-local-dev-proxy/pull/1/commits/c57fc32b825176bdf278a1645faeea254baeb0cb). Will Update the doc link once docs are published.

File: server.js (hunk 1 above)
Comment (ThumulaPerera) [reply server.js:89]: Resolved with [Add proper logger](https://github.com/wso2/choreo-managed-auth-local-dev-proxy/pull/1/commits/fafa91da09924d89d13b0b80d2da25071050b1bd)

**Review by senthalan — APPROVED** [review]

**Pull request merged**